### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by sinumohan**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - `git checkout -b feature-branch`
+- Make the required changes
+- Test the changes by using below command
+  - `yarn` (Installs all the dependencies)
+  - `yarn test`
+- Create a pull request using below commands
+  - `git add --all`
+  - `git commit -m "your commit message"`
+  - `git push origin feature-branch`
+- Go to [Repository](https://github.com/arshadkazmi42/rhof/)
```
Comment (arshadkazmi42) [inline CONTRIBUTING.md]: change this url

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+- Create Pull Request against `master` branch
+- Add a suitable title and description to the pull request and tag the issue number in Pull Request description, if the pull request is related to some issue logged here: [Issues](https://github.com/arshadkazmi42/rhof/issues)
```
Comment (arshadkazmi42) [inline CONTRIBUTING.md]: change this url

File: CONTRIBUTING.md (hunk 1 above)
Comment (sinumohan) [reply CONTRIBUTING.md]: done

File: CONTRIBUTING.md (hunk 2 above)
Comment (sinumohan) [reply CONTRIBUTING.md]: done

**Pull request merged**